Code/Tools/ProjectManager/Source/GemCatalog/GemInspector.cpp

```diff
@@ -236,6 +238,26 @@ namespace O3DE::ProjectManager
             connect(m_versionComboBox, QOverload<int>::of(&QComboBox::currentIndexChanged), this, &GemInspector::OnVersionChanged);
         }
 
+        m_compatibilityTextLabel->setVisible(!gemInfo.IsCompatible());
+        if(!gemInfo.IsCompatible())
+        {
+            if(!gemInfo.m_compatibleEngines.isEmpty())
+            {
+                if (m_readOnly)
+                {
+                    m_compatibilityTextLabel->setText(tr("This version is not known to be compatible with the current engine"));
+                }
+                else
+                {
+                    m_compatibilityTextLabel->setText(tr("This version is not known to be compatible with the engine this project uses"));
+                }
```

Comment:
How does the read only option change which engine the project isn't associated with?

Reply:
I should add a comment in the code - in read only mode there is no project being configured, you are looking at all gems available via the engine and o3de manifest so I tried to make the error message not mention "project"